attacker/attacker.go

```diff
@@ -123,16 +124,19 @@ func RunVegeta(ctx context.Context, requestDicts []map[string]interface{}, testN
 	report := vegeta.NewTextReporter(&metrics)
 	err := report.Report(os.Stdout)
 	if err != nil {
-		zlog.Debug(ctx).Msg("vegeta report command failure")
+		zlog.Info(ctx).Msg("vegeta report command failure")
 		return err
 	}
-	zlog.Debug(ctx).Msg("Vegeta attack completed successfully")
+	zlog.Info(ctx).Msg("Vegeta attack completed successfully")
+	endTime := time.Now()
+	elapsedTime := endTime.Sub(startTime)
+	zlog.Info(ctx).Msg(fmt.Sprintf("Total time taken for %s: %02d hours %02d minutes %02d seconds", testName, int(elapsedTime.Hours()), int(elapsedTime.Minutes())%60, int(elapsedTime.Seconds())%60))
 
 	// Indexing results to elastic search
 	if attackMap["ESHost"] != "" && attackMap["ESPort"] != "" && attackMap["ESIndex"] != "" {
 		err = indexVegetaResults(ctx, metrics, testName, attackMap)
 		if err != nil {
-			zlog.Debug(ctx).Msg("Failed to indexing results to elastic search")
+			zlog.Info(ctx).Msg("Failed to indexing results to elastic search")
```

Comment:
Same here

Reply:
Changing this log level to Debug, as its an error case.